torch/optim/lr_scheduler.py

```diff
@@ -603,6 +603,44 @@ def _get_closed_form_lr(self):
                 for base_lr in self.base_lrs]
 
 
+class CombinedScheduler(object):
```

Comment:
Not sure if we should inherit from `_LRScheduler` but I think we need to implement `state_dict` and `load_state_dict` so that existing scripts will continue working without issues. See [here](https://github.com/pytorch/vision/blob/master/references/video_classification/train.py#L220) and [here](https://github.com/pytorch/vision/blob/master/references/video_classification/train.py#L239). I'm not sure if also implementing `get_last_lr`, `get_lr` and `print_lr` makes sense or not. Thoughts?

Reply:
I agree with @datumbox , I think making `CombinedScheduler` inherit from `_LRScheduler` would probably be more natural, as it effectively be a drop-in replacement for individual schedulers.
So `CombinedScheduler([scheduler])` would be semantically the same as `scheduler` for all operations that `scheduler` supports.

But this might come with some challenges,